etna/analysis/plotters.py

```diff
@@ -1099,13 +1100,37 @@ def plot_trend(
         ax[i].legend()
 
 
+def get_fictitious_relevances(pvalues: pd.DataFrame, alpha: float) -> Tuple[np.ndarray, float]:
+    """
+    Convert p-values into fictitious variables, with function f(x) = 1 - x.
```

Comment:
Add blank line after this line. Below we have extended description that should be separated.